todo_app/data/trello_items.py

```diff
@@ -0,0 +1,68 @@
+import requests
+import os
+from todo_app.data.Item import Item
+
+
+def get_lists():
+    url = "https://api.trello.com/1/boards/" + os.getenv('BOARD_ID') + "/lists"
```

Comment:
Please can you make a helper method called `build_url` which accepts the unique part of the URL and prepends the common base `https://api.trello.com/1/` part?

That way, to update the base URL we would only need to change it in one place (e.g. if we were to upgrade from version 1 of the API to version 2).